schemas/iso19139/src/main/plugin/iso19139/process/sds-add-dataquality.xsl

```diff
@@ -0,0 +1,92 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Would be good on new file to set the copyright too if you've time
See https://github.com/geonetwork/core-geonetwork/blob/develop/schemas/iso19139/src/main/plugin/iso19139/process/anonymizer.xsl#L2

The copyright template is here https://github.com/geonetwork/core-geonetwork/blob/develop/docs/copyright.txt and IDE usually have a menu to auto populate missings.